### DISCUSSION TITLE
XXL:  Kim-2 settings

### DISCUSSION THREAD

**Discussion by dgoryeo** [Purfview/whisper-standalone-win#238] [Q&A] [answered]
Brilliant work @Purfview with XXL! The added features are greatly useful. I was so excited when I saw you have added Kim-2 and Silero-3. Very good choices. I just tested XXL beta running on RTX3600 with very good results.

A quick question: in the past I have used Kim-2 in UVR5. In UVR5 the setting prameters for Kim-2 (MDX) are: Segment Size, and Overlap.  In XXL beta the (available) setting parameter is mdx_chunk. I was wondering (a) does mdx_chunk somehow correspond to the segments_size? (b) if available, would it be an idea to expose the overlap parameter in CLI?

**Comment by Purfview** [comment, marked as answer]
I meant new post(aka comment) instead of reply, not new thread. :)

>A quick question: in the past I have used Kim-2 in UVR5. In UVR5 the setting prameters for Kim-2 (MDX) are: Segment Size, and Overlap. In XXL beta the (available) setting parameter is mdx_chunk. I was wondering (a) does mdx_chunk somehow correspond to the segments_size? (b) if available, would it be an idea to expose the overlap parameter in CLI?

"Segment Size" is related to mdx_chunk, here I think it corresponds to seconds, there I dunno what it means, as I remember there Segment Size default is "256" and it corresponded to ~5-8(?) seconds.
Here is no such thing as "Overlap", it's not direct implementation from UVR.

**Reply by Purfview** [reply]
Btw, Kim_vocal_2 is pretty good but current the best vocal extraction is BS RoFormer model, I'll check if that can be implemented.
